.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
 
         strategy:
             matrix:
-                node-version: [ 16.13.x, 18.10.x ] # Include version 18 once upgraded to Angular 15
+                node-version: [ 16.14.x, 18.10.x ] # Include version 18 once upgraded to Angular 15
```

Comment:
since you changed 16.13.x to 16.14.x, you have to adapt the required checks for the main branch.